NavitiaSDKUI/LocaliseModule.rb

```diff
@@ -0,0 +1,139 @@
+          puts "Extracting #{entry_path}"
+
+          unless File.directory?(localise_directory)
+            FileUtils.mkdir_p(localise_directory)
+            zip_file.extract(entry, entry_path) unless File.exist?(entry_path)
+            zipFiles << entry_path
+          end
+        end
+      end
+    end
+    return zipFiles
+  end
+
+=begin
+Download Zip file using localise.biz REST API
+Parameters:
+- +tag+::         +string+  tag used in localise.biz
+=end
+  def download_localise_files(tag)
+    if !(ENV['LOCALISE_BIZ_USER'].presence && ENV['LOCALISE_BIZ_PASSWORD'].presence)
+        puts "❌ WHO ARE YOU? 👁👁. ❌"
+    else 
+      uri = URI.parse("https://localise.biz/api/export/archive/strings.zip?filter=#{tag}&fallback=en-US&order=id")
+      request = Net::HTTP::Get.new(uri)
+      request.basic_auth(ENV['LOCALISE_BIZ_USER'], ENV['LOCALISE_BIZ_PASSWORD'])
+      request.content_type = "application/zip"
+      req_options = {
+          use_ssl: uri.scheme == "https",
+      }
+  
+      response = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
+        http.request(request)
+      end
+  
+      if response.code == "200"
+        puts "Download Request from localise.biz -> RESPONSE: #{response.code}"
+        File.open(ZIP_FILE_NAME, "w") do |file|
+          file.write response.body
+        end
+        return true
+      else 
+        puts "Download Request FAILED -> RESPONSE: #{response.code}"
+        return false
+      end
+    end
+  end
+
+  def add_localizable_to_project(list_of_localizable_path)
+    project_path = './NavitiaSDKUI.xcodeproj'
+    project = Xcodeproj::Project.open(project_path)
+    
+    for o in project.objects do 
+      if o.is_a? Xcodeproj::Project::Object::PBXGroup
+          if o.hierarchy_path == "/NavitiaSDKUI/Resources/Localizable.strings"
+            puts "Localizable group found! That's good news!"
+            group = o
+          break
+        end
+      end
+    end
+    
+    existing_localizable_in_project = group.files.map { |file| file.name }
+    for localizable_path in list_of_localizable_path do 
+      localizable_code = localizable_path.split('.').first
+      if !existing_localizable_in_project.include?(localizable_code)
+        puts "Localizable does not existing in project! Let's add it!".
+        file = project.new_file(localizable_path)
+        file.move(group)
+        file.name = localizable_code
+      end
+    end
+    project.save
+  end
+
+  def copy_to_project(raw_zip_files) 
+    raw_zip_files.each do |file|
+      language_marker = "-#{file.split("-").last.split(".").first}"
+      language_path = file.gsub(language_marker, "")
+      FileUtils.cp_r(File.dirname(language_path), RESOURCES_PATH)
+    end
+  end
+
+  def cleaning(raw_zip_files)
+    for filepath in raw_zip_files do
+      FileUtils.rm_rf(File.dirname(filepath))
+      File.delete(ZIP_FILE_NAME) if File.exist?(ZIP_FILE_NAME)
+    end
+  end
```

Comment:
I made a mistake in this function. 
Can you replace it with this code :  
```suggestion
  def cleaning(raw_zip_files)
    for filepath in raw_zip_files do
      FileUtils.rm_rf(File.dirname(filepath))
      File.delete(ZIP_FILE_NAME) if File.exist?(ZIP_FILE_NAME)
    end
  end
```